### PR TITLE
Added DarQ icon

### PR DESCRIPTION
**Pull request by LimeWools**

### CODE REVIEW

**Comment by arun54321** [comment]
That svg looks very complex to me. What do you think about this?

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
  <path style="fill:#000" d="M12 2C6.486 2 2 6.486 2 12s4.486 10 10 10 10-4.486 10-10S17.514 2 12 2zm0 1.533A8.456 8.456 0 0 1 20.467 12 8.454 8.454 0 0 1 12 20.465V3.533z"/>
</svg>
```

**Comment by LimeWools** [comment]
@arun54321 Yeah that looks a lot better than the code that was a part of the svg.

**Comment by paphonb** [comment]
> That svg looks very complex to me. What do you think about this?
> 
> ```
> <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
>   <path style="fill:#000" d="M12 2C6.486 2 2 6.486 2 12s4.486 10 10 10 10-4.486 10-10S17.514 2 12 2zm0 1.533A8.456 8.456 0 0 1 20.467 12 8.454 8.454 0 0 1 12 20.465V3.533z"/>
> </svg>
> ```
> 
>  ```
> <vector xmlns:android="http://schemas.android.com/apk/res/android"
>     android:width="24dp"
>     android:height="24dp"
>     android:viewportWidth="24"
>     android:viewportHeight="24">
> 
>     <path
>         android:fillColor="#000"
>         android:pathData="M12 2C6.486 2 2 6.486 2 12s4.486 10 10 10 10-4.486 10-10S17.514 2 12 2zm0
> 1.533A8.456 8.456 0 0 1 20.467 12 8.454 8.454 0 0 1 12 20.465V3.533z" />
> </vector>
> ```

I agree. @LimeWools can you change to his version instead?

**Comment by LimeWools** [comment]
@paphonb Sure will do that.

File: app/src/main/res/drawable/darq.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <path
+        android:fillColor="#000"
+        android:pathData="M12 2C6.486 2 2 6.486 2 12s4.486 10 10 10 10-4.486 10-10S17.514 2 12 2zm0
+1.533A8.456 8.456 0 0 1 20.467 12 8.454 8.454 0 0 1 12 20.465V3.533z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/darq.xml:11]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="M12,2A10,10 0,0 0,2 12A10,10 0,0 0,12 22A10,10 0,0 0,22 12A10,10 0,0 0,12 2zM12,4A8,8 0,0 1,20 12A8,8 0,0 1,12 20L12,4z"
      android:strokeWidth="2"
      android:fillColor="#000000"
      android:strokeLineCap="round"/>
</vector>
```

File: svgs/darq.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path style="fill:#000" d="M12 2C6.486 2 2 6.486 2 12s4.486 10 10 10 10-4.486 10-10S17.514 2 12 2zm0 1.533A8.456 8.456 0 0 1 20.467 12 8.454 8.454 0 0 1 12 20.465V3.533z"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/darq.svg:3]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   viewBox="0 0 24 24"
   version="1.1"
   id="svg4"
   sodipodi:docname="darq.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs8" />
  <sodipodi:namedview
     id="namedview6"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="true"
     inkscape:zoom="22.892582"
     inkscape:cx="7.7317622"
     inkscape:cy="15.048543"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg4">
    <inkscape:grid
       type="xygrid"
       id="grid1826"
       empspacing="1" />
  </sodipodi:namedview>
  <path
     id="path1283"
     style="opacity:1;fill:#000000;stroke-width:2;stroke-linecap:round"
     d="M 12 2 A 10 10 0 0 0 2 12 A 10 10 0 0 0 12 22 A 10 10 0 0 0 22 12 A 10 10 0 0 0 12 2 z M 12 4 A 8 8 0 0 1 20 12 A 8 8 0 0 1 12 20 L 12 4 z " />
</svg>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
I’d suggest making the circular outline a bit thicker.

**Comment by LimeWools** [comment]
@patrykmichalik Updated the .xml and svg files as requested.

**Comment by patrykmichalik** [comment]
Is the pull request ready to merge, @LimeWools? You seem to have marked it as a draft. 🤔

**Comment by LimeWools** [comment]
@patrykmichalik  Oh sorry about that yes it's ready.

**Comment by patrykmichalik** [comment]
No worries! I’ll merge it, then. Thanks for contributing!

**Review by patrykmichalik — APPROVED** [review]

**Comment by alaamokhimar** [comment]
good

**Pull request merged**